test/unittests/execute_test.cpp

```diff
@@ -1048,3 +1048,221 @@ TEST(execute, memory_copy_32bytes)
     std::copy_n(&(*instance.memory)[33], input.size(), std::back_inserter(output));
     EXPECT_EQ(output, input);
 }
+
+TEST(execute, fp_instructions)
+{
+    /* wat2wasm
+    (memory 1)
+
+    (func (param f32 f64)
+      ;; f32 ops
+      i32.const 0 ;; mem offset
+      f32.const 1
+      ;; get_local 0 -- this should work
```

Comment:
Anyone want to investigate? I think the local calculation may be wrong and this indexes out of bounds.

Reply:
You pass empty `args` vector to `execute`, so it's out of bounds

Reply:
Phew, rookie mistake :)

Reply:
But I think I'll just remove it because we can't pass floats properly yet.